rest-api-spec/test/indices.get_aliases/10_basic.yaml

```diff
@@ -202,7 +202,7 @@ setup:
 "Getting alias on an non-existent index should return 404":
 
   - skip:
-        version: 1 - 999
+        version: 0 - 999
```

Comment:
This happened from @rjernst refactoring of version logic in rest tests. I don't know what these are doing personally.